plugins/org.obeonetwork.m2doc.doc/doc/user/M2Doc User Guide.textile

```diff
@@ -1,344 +1,357 @@
+
+You will have to set those two URIs in the Package NSURI attribute: 
+
+* http://www.obeonetwork.org/dsl/database/1.0
+
+* http://www.obeonetwork.org/dsl/typeslibrary/1.0
+
+If you are in a Modeling project with a database model and have installed M2Doc Sirius extension, these two URIS are set automatically. 
+
+and the result file name.
+
+The resulting property sheet should look like this:
+
+!images/image007.png!
+
+
+h3. Configure variables definitions
+
+Now, you have to configure the variables definitions. If these variables are set in the template file, they are automatically added to the configuration model and you just need to value them.
+
+Otherwise, you have to create these variables definitions.
+
+
+h4. Manage variables with M2Doc Sirius viewpoint
+
+
+ A Sirius viewpoint __M2Doc__  is activated on the __org.obeonetwork.m2doc.dbexample__  example project.  In the configuration diagram on Generation element, you can manage the variables definitions.
+
+Two kind of variable definitions can be created:
+
+* string definition: to add one, use the palette tool and set the key and the value (direct edit or properties view)
+
+!images/image008.png!
+
+ 
+
+*  model definition: to add one, select the model element value in the model explorer and drag n drop it into the diagram and set the key name (direct edit or property view).
+
+You can also use the palette tool and select the model element value.
+
+!images/image009.png!
+
+
+h4. Create variable with EMF Editor
+
+If you do not have Sirius, you have to set the variable via the genconf EMF Editor:
+
+First, you have to load the main database resource: 
+
+ !images/image010.png!
+
+Select the database model that is in the model directory:
+
+!images/image011.png!
+
+Now that the database resource is loaded, we will be able to use model element in it as variable's values. 
+
+We first create a string variable, key='author', value='John Doe':
+
+!images/image012.png!
+
+And a model variable:
+
+!images/image013.png!
+
+Set the values of the variables in the property view. The model variable must be defined like follows:
+
+!images/image014.png!
+
+
+h3. Generate documentation
+
+You just have to invoke the generation by right clicking on the generation model element:
+
+!images/image015.png!
+
+A message indicates everything went well:
+
+!images/image016.png!
+
+Refresh the 'doc' directory if needed. The result should be in it. 
+
+
+h3. Template validation
+
+When the generation is launched, we first run a template validation. If this validation gets errors, an error file will be generated near the generated result file.
+
+The template validation can also be started alone, if it gets errors, an error file will be generated near the template file.
+
+!images/image017.png!
+
+The template validation checks:
+
+*  M2Doc syntax,
+
+*  AQL expression syntax and and typing,
+
+* Feature and services called in AQL expressions exist,
+
+* Variables syntax and typing.
+
+ 

```

Comment:
It's very hard to understand what has changed... Could you go over all the files that have CRLF problems an use the original CRLF separator so that the differences are less numerous?